docs/troubleshooting/assisted_installer_failing_to_fetch_iso.md

```diff
@@ -1,16 +1,20 @@
-If you are seeing entries like the one below in the logs of `installer` container:
+# Failed To Upload Boot Files
 
-```
-level=fatal msg="Failed to upload boot files" func=main.main.func1 file="/go/src/github.com/openshift/origin/cmd/main.go:191" error="Failed uploading boot files for OCP version 4.8: Failed fetching from URL https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.14/rhcos-4.8.14-x86_64-live.x86_64.iso: Get \"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.14/rhcos-4.8.14-x86_64-live.x86_64.iso\": dial tcp: i/o timeout"
-```
-Then you should check if your containers can resolve `mirror.openshift.com`.
-It could be that the hosts dns config is invalid for the container (e.g. 127.0.0.1).
-An easy way to do this if you are using the `http_store` is run:
+If you are seeing errors in the container logs like the one below from the `installer` container:
 
-$ podman run -it httpd-24-centos7 /bin/bash
-> $ dig mirror.openshift.com
-> $ dig @8.8.8.8 mirror.openshift.com
-> $ dig @8.8.4.4 mirror.openshift.com
+> level=fatal msg="Failed to upload boot files" func=main.main.func1 file="/go/src/github.com/openshift/origin/cmd/main.go:191" error="Failed uploading boot files for OCP version 4.8: Failed fetching from URL https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.14/rhcos-4.8.14-x86_64-live.x86_64.iso: Get \"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.14/rhcos-4.8.14-x86_64-live.x86_64.iso\": dial tcp: i/o timeout"
+
+
+To check if the DNS for your container is configured and can resolve `mirror.openshift.com`.
+Try the following:
+  1. Connect to the containers shell.
+      > $ podman run -it ${container_id} /bin/bash
```

Comment:
Is `$container_id` the id of the installer container? It need to state that or people are likely to get confused.

We can probably add a line to grab that and store it in a var to make it even more straight forward.

Reply:
I can expand it by adding the `podman ps` command and showing sample output.